zerver/openapi/zulip.yaml

```diff
@@ -8310,7 +8324,12 @@ paths:
         of the [`realm op: update_dict`](/api/get-events#realm-update_dict)
         event in [`GET /events`](/api/get-events).
 
-        **Changes**: In Zulip 10.0 (feature level 316), `edit_topic_policy`
+        **Changes**: Prior to Zulip 10.0 (feature level 319), the permission to
+        resolving a topic was managed by `can_move_messages_between_topics_group`.
```

Comment:
"permission for resolving" would be better grammar.

Reply:
oops, ya!